### PR TITLE
Allow searching by city or town

### PR DESCRIPTION
**Pull request by mwellman17**
Allow searching by city or town.

Geocoder already supports searching by any part of an address.

The Geocoder has component filtering by country and locality (like zip code), but no hard filter exists for state.
Adding ' Massachusetts' to the end of the address string provides an adequate filter.

Proof of concept:
Before the add, searching "Lawrence" returns Lawrence Kansas. The returned search results are Lee, Pittsfield, and others in the western most part of the state.
After the add, searching "Lawrence" returns results in Haverhill and other sites near Lawrence Mass.

What if a user enters the state as well?
Tried entering towns plus "MA", "Mass", "Massachusetts", and spelling errors. The Geocoder seems very forgiving and the results were consistently good across several towns all over the state.

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -11,26 +11,30 @@ class Search extends React.Component {
...
         siteData: [],
-        zipCode: '',
+        address: '',
         availability: 'Sites with reported doses',
-        zipCodeError: false,
+        addressError: false,
         geolocationError: false
     }
 
     searchByZipCode = () => {
```
Comment (hblumberg) [inline pages/search.js]: Nit: update function name

File: pages/search.js [hunk 2]
```diff
@@ -143,11 +142,11 @@ class Search extends React.Component {
                                 <button id="geolocate" onClick={this.searchByGeolocation} className="btn btn-primary">My Location</button>
                                 {this.state.geolocationError && <p>Cannot figure out your location.</p>}
                                 <div className="form-group">
-                                    <label htmlFor="zipCode">or near</label>
-                                    <input type="text" className="form-control" id="zipCode" name="zipCode" placeholder="5-digit zip code" required="" value={this.state.zipCode} onChange={this.handleChange} />
+                                    <label htmlFor="address">or near</label>
+                                    <input type="text" className="form-control" id="address" name="address" placeholder="city, town, or zip code" required="" value={this.state.address} onChange={this.handleChange} />
```
Comment (hblumberg) [inline pages/search.js]: Nit: capitalize city maybe?

**Review by hblumberg — APPROVED** [review]
This is great! Glad the API is so forgiving 😅

File: pages/search.js (hunk 1 above)
Comment (mwellman17) [reply pages/search.js]: Yes. Made that change in mw/enter which I want to merge into this branch before merging with main.

File: pages/search.js (hunk 2 above)
Comment (mwellman17) [reply pages/search.js]: good call

**Pull request merged**